### ISSUE TITLE
State

### ISSUE THREAD

**Issue by simcha-c** [asallem94/Meetin#2]
Hey Sallem :)

1. Put all of the state into the 3 backticks to show it all as a code snippet.
2. The users slice shouldn't be capitalized.
3. A user will be a member of many groups, and so in every user object - we would want an array with group ids that the user is a part of.
4. A user rsvps to many events. If we will show that anywhere on the website, we would want an array of the event ids the user is going to.
5. A user can also create a bunch of groups. so we need a created group ids array. If we would show that on the website.
6. A user can also create many events. if we're showing that anywhere, it also needs to be in the user object info.
7. Take out the interests slice of state. You'll add that in when you start working on that feature.
Basically the state should have 4 slices in the entities slice of state. Users, groups, and events. We'll do comments later on. Each slice should have reference to ids of what they have many of etc. 

Work on fixing the state, and i'll look at it again once you fix it.
